docker/docker.mk

```diff
@@ -179,7 +171,7 @@ docker-windows-base-push: check-windows-version
 DEV_CONTAINER_VERSION_TAG?=0.1.5
 
 # Use this to pin a specific version of the Dapr CLI to a devcontainer
-DEV_CONTAINER_CLI_TAG?=1.5.0
+DEV_CONTAINER_CLI_TAG?=1.5.1
```

Comment:
Hang on, did we have a CLI point fix?

Reply:
Yes

Reply:
Oh, we did have one, it just hadn't been marked as release in the UI. Fixed